package/network/services/picomdnsd/files/picomdnsd.init

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+USE_PROCD=1
+
+APPBINARY=/usr/sbin/picomdnsd
+LANIPADDR=$(/sbin/ifconfig "br-lan" | grep "inet " | awk -F' ' '{print substr($2,6)}')
+
+start_service() {
+	procd_open_instance
+	procd_set_param command "$APPBINARY" openwrt "$LANIPADDR"
```

Comment:
Dont you want the `hostname` as found in UCI `system` with default openwrt only if empty?

Reply:
That's a good idea, would this be a good idea?
```uci get system.@system[0].hostname```